FunctionalTableDataDemo/AppDelegate.swift

```diff
@@ -7,13 +7,19 @@
 //
 
 import UIKit
+import FunctionalTableData
 
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
 
 	var window: UIWindow?
 
 	func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+		
+		Separator.appearance().backgroundColor = UITableView().separatorColor
```

Comment:
initializing a tableview to get the separate colour? 🤢

Reply:
Is the only way to get the real default value 😔 but I will use a random Gray color instead so this idea doesn't spread